setup.py

```diff
@@ -97,6 +100,7 @@ def get_version(version_file):
     ],
     extras_require={
         "test": [
+            "boto3-stubs",
```

Comment:
Are these typing packages actually necessary for testing? The `mypy` package is only installed as part of the development requirements so I think these should go in `requirements-dev.txt`.

Reply:
Makes sense - please see https://github.com/cisagov/assessment-data-export/pull/46/commits/799661f7438cdb6f782150e49d9079aecf11a140.